bee-message/src/output/alias.rs

```diff
@@ -123,6 +120,19 @@ impl AliasOutputBuilder {
         self
     }
 
+    ///
+    pub fn replace_unlock_condition(mut self, unlock_condition: UnlockCondition) -> Self {
```

Comment:
Would `add_or_replace_unlock_condition` be a better name?

Reply:
what about `upsert_unlock_condition`? too niche?

Reply:
yea, I like that. Keeps the method name short as before and addresses my suggestion.

Reply:
Hmm I think it's really niche tbh, I didn't even know this term a couple of weeks ago...

Reply:
I din't have an issue with long method names